buffer/src/main/java/io/netty/buffer/ByteBufUtil.java

```diff
@@ -125,6 +126,13 @@ public static String hexDump(byte[] array, int fromIndex, int length) {
     }
 
     /**
+     * Decodes a string generated by {@link #hexDump(byte[])}
+     */
+    public static byte[] decodeHexDump(String hexDump) {
```

Comment:
The method was meant to be symmetrical with `hexDump` which produces a `String` ... but since the implementation doesn't depend on `String` I can change to `CharSequence`